readme-vars.yml

```diff
@@ -33,36 +33,39 @@ param_env_vars:
 opt_param_usage_include_env: true
 opt_param_env_vars:
   - { env_var: "BASE_URL", env_value: "", desc: "Specify the full URL (including protocol) when running behind a reverse proxy" }
+  - { env_var: "PLAYWRIGHT_DRIVER_URL", env_value: "", desc: "Specify the full URL to your chrome driver instance. See the [wiki](https://github.com/dgtlmoon/changedetection.io/wiki/Playwright-content-fetcher) for details." }
 
 custom_compose: |
   ---
-  version: "2.1"
   services:
-    changedetection:
+    changedetection.io:
```

Comment:
We may want to add a ram limit to the example due to the potential memory leak
[https://github.com/dgtlmoon/changedetection.io/wiki/Playwright-content-fetcher#playwright-memory-leak](https://github.com/dgtlmoon/changedetection.io/wiki/Playwright-content-fetcher#playwright-memory-leak)

Reply:
It should now be mitigated, if not completely fixed, so it won't grow endlessly any more